docs/export-module/deprecated-async-option.md

```diff
@@ -0,0 +1,65 @@
+Deprecating the export server async option
+------------------------------------------
+
+Version 2.2 of the Highcharts Node.js Export Server, due December 1st 2021, will deprecate the `async` option. The reasons for that is that it makes the application dependent on saving state, and complicates load balancing. Furthermore it is no longer needed in browsers because we can render the returned image directly as a base64 encoded `src`.
+
+Here's how to migrate from the `async` option.
+
+### Browser
+
+In the browser, the config can be posted and the returns be rendered using the FormData, fetch and FileReader API's. See the [example on jsFiddle](https://jsfiddle.net/highcharts/mt2s95Ln/) for how to set this up with an async-await based syntax.
+
+If legacy browsers are are a requirement, see also the [old jQuery based demo](https://jsfiddle.net/highcharts/edyucqkw/). Note that this approach uses the export server's `b64` option to do the base64 encoding on the server side.
+
+### Node
+
+If you're running a node environment, the recommended approach would be to use the export server as a node module instead of posting content to Highsoft's online server. For details, see the chapter [Using as a Node.js Module](https://github.com/highcharts/node-export-server/blob/master/README.md#using-as-a-nodejs-module) in the project repo.
```

Comment:
capitalize Node? Or maybe Node.js?

Reply:
Thanks! Node.js is the trademark they're using.